vignettes/EnrichmentBrowser.Rmd

```diff
@@ -0,0 +1,989 @@
+    two sample groups, optionally including paired samples or sample
+    batches. Therefore, we recommend the highly customizable SAFE for
+    testing a directional hypothesis or in situations of more complex
+    experimental designs such as comparisons between multiple groups,
+    continuous phenotypes or the presence of covariates.
+
+See also Reference [@Geistlinger20] for the results of the benchmarking
+study and the `r Biocpkg("GSEABenchmarkeR")` package for a general framework for reproducible benchmarking of gene set enrichment methods.
+
+### Microarray data
+
+Given normalized log2 intensities for the ALL microarray dataset, a
+basic ORA can be carried out via
+
+```{r sbea}
+sbea.res <- sbea(method = "ora", se = allSE, gs = hsa.gs, perm = 0, alpha = 0.1) 
+gsRanking(sbea.res)
+```
+
+Note that we set `perm = 0` to invoke the classical hypergeometric test without sample permutation, and that we chose a significance level $\alpha$ of 0.1 for demonstration purposes.
+
+### RNA-seq data
+
+When analyzing RNA-seq datasets with expression values given as logTPMs
+(or logRPKMs / logFPKMs), the available set-based enrichment methods can
+be applied as for microarray data. However, when given raw read counts
+as for the airway dataset, we recommend to first apply a
+variance-stabilizing transformation such as `voom` to arrive at library-size normalized logCPMs.
+
+```{r vst}
+ airSE <- normalize(airSE, norm.method = "vst")
+```
+
+The mean-variance relationship of the transformed data is similar to
+what is observed for microarray data, simplifying the application of
+legacy enrichment methods such as GSEA and PADOG to RNA-seq data, and
+enable the use of fast and established methods.
+
+```{r gsea-rseq, eval=FALSE}
+air.res <- sbea(method = "gsea", se = airSE, gs = hsa.gs) gsRanking(sbea.res)
+```
+
+## Result exploration
+
+The result of every enrichment analysis is a ranking of gene sets by the
+corresponding *p*-value. The `gsRanking` function displays by default only those gene sets satisfying the chosen significance level $\alpha$, but we can use to obtain the full ranking.
+
+```{r fullrank}
+ gsRanking(sbea.res, signif.only = FALSE)
+```
+
+
+While such a ranked list is the standard output of existing enrichment
+tools, the package `r Biocpkg("EnrichmentBrowser")` provides visualization and interactive exploration of resulting gene sets far beyond that point. Using the `eaBrowse` function creates a HTML summary from which each gene set can be inspected in detail (this builds on functionality from the `ReportingTools` package).
+
+The various options are described in Figure \@ref(fig:oraRes).
+
+```{r eaBrowse, eval=FALSE}
+ eaBrowse(sbea.res)
+```
+
+
+![(#fig:oraRes) ORA result view 
+.For each significant gene set in the ranking, the user can select to view (1) a gene report, that lists all genes of a set along with fold change and DE p-value, (2) interactive overview plots
```

Comment:
Remove period before at the beginning of the sentence.

Reply:
k